### DISCUSSION TITLE
footprint depth per cell

### DISCUSSION THREAD

**Discussion by mitchTwoTimes** [GreenleafLab/ArchR#1089] [Questions / Documentation]
Is there a way to get footprint depth out of the `getFootprints()` output? Ideally I could have colData attached to these values.

Thanks

**Comment by rcorces** [comment]
we dont provide these metrics but I believe you can calculate them from the values stored in the output of `getFootprints()`?

**Reply by mitchTwoTimes** [reply]
Thanks for your response. What would that look like? Is there a resource where I could learn how to do this?

**Reply by rcorces** [reply]
Sorry - this is outside the scope of the support we provide. You can read Baek et al 2017 (BaGFoot) which might provide some hints. 
https://pubmed.ncbi.nlm.nih.gov/28538187/
You might also find some hints in the Sup Material from our 2018 Science paper - https://www.science.org/doi/10.1126/science.aav1898
